src/VirtualClient/VirtualClient.Main/profiles/PERF-GPU-MLPERF-NVIDIA-H100.json

```diff
@@ -0,0 +1,93 @@
+{
+    "Description": "MLPerf GPU Performance Workload",
+    "Metadata": {
+        "RecommendedMinimumExecutionTime": "10:00:00",
+        "SupportedPlatforms": "linux-x64",
+        "SupportedOperatingSystems": "Ubuntu",
+        "SpecialRequirements": "This is an Nvidia GPU-specialized workload. It depends upon the system having an Nvidia GPU card/chip.",
+        "GPU": "H100"
```

Comment:
Is this referenced? Why do we need a separate profile for H100?

Reply:
List of benchmarks supported by MLPerf for A series and then H series is different. This brings a good point that i should rename to Hseries instead of H100. To keep things clear for user, I have created a separate profile. I know that this is not a sustainable approach but in my opinion we can keep MLPerf in our exceptional benchmarks bucket that needs attention with new GPU HW coming in. As of now, quite a few people are interested and will be interested to run MLPerf for H100 and then probably B200 and then later do comparisons among different, so to keep things simple to run, I have created separate profiles for them.